invenio_rdm_records/fixtures/tasks.py

```diff
@@ -27,3 +28,49 @@ def create_demo_record(data):
     service = current_rdm_records.records_service
     draft = service.create(data=data, identity=system_identity)
     service.publish(id_=draft.id, identity=system_identity)
+
+
+@shared_task(ignore_result=True)
+def update_expired_embargos():
+    """Release expired embargoes every midnight."""
+
+    def lift_embargo_from_draft(service, rdm_record):
+        """Lifts embargo from draft."""
+        record_draft = service.edit(rdm_record["id"], system_identity)
+        draft = record_draft._record
+        if draft.access.embargo.lift():
```

Comment:
So something like:
```python
if draft.access.embargo.lift():
    service.update_draft(...)

```

We might need to add a service layer function, to avoid accessing the `_record` here (it's whats receiving as parameter)

Reply:
After discussion decided to implement `lift` functionality in the service.